test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -276,6 +276,15 @@ TEST_F(AtenXlaTensorTest, TestSubScalarInPlace) {
   });
 }
 
+TEST_F(AtenXlaTensorTest, TestSymSizes) {
```

Comment:
nit, `TestSymSizesStatic`?

Reply:
Name is ok now given it tests both dynamic and static `SymSizes`